app/models/form_profile.rb

```diff
@@ -336,12 +324,10 @@ def vet360_contact_info
     return @vet360_contact_info if @vet360_contact_info_retrieved
 
     @vet360_contact_info_retrieved = true
-    if Flipper.enabled?(:remove_pciu, user) && user.icn.present?
+    if user.icn.present?
```

Comment:
Ensure ICN is present before initializing vet360_contact_info. ICN is required for vet360_contact_info